src/stories/containers/TransparencyReport/components/TransparencyForecast/useTransparencyForecast.tsx

```diff
@@ -45,6 +50,22 @@ export const useTransparencyForecast = (currentMonth: DateTime, budgetStatements
     return _.sortBy(Object.values(dict), 'id');
   }, [currentMonth, budgetStatements]);
 
+  const getTransferRequestTargetBalanceColumn = useCallback((wallet: BudgetStatementWalletDto) => {
+    const targetWithTimeSpan: Pick<BudgetStatementWalletTransferRequestDto, 'target' | 'walletBalanceTimeStamp'> =
+      {} as BudgetStatementWalletTransferRequestDto;
+
+    const lastIndex = (wallet.budgetStatementTransferRequest ?? []).length - 1;
+    if (wallet.budgetStatementTransferRequest && wallet.budgetStatementTransferRequest.length > 0) {
+      targetWithTimeSpan.target = wallet.budgetStatementTransferRequest[lastIndex].target;
+      targetWithTimeSpan.walletBalanceTimeStamp =
+        wallet.budgetStatementTransferRequest[lastIndex]?.walletBalanceTimeStamp;
+    }
+
+    return targetWithTimeSpan;
+  }, []);
+
+  const headersValuesToolTip = getTransferRequestTargetBalanceColumn(wallets[0]);
```

Comment:
as this is used in the `mainTableColumns` only, you could move this inside the `useMemo` hook of the `mainTableColumns` calculation so you avoid to make this calculation on every render